<ui version="4.0" >
 <class>KWinAdvancedCompositingOptions</class>
 <widget class="QWidget" name="KWinAdvancedCompositingOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>326</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QFormLayout" name="formLayout" >
     <property name="fieldGrowthPolicy" >
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Compositing type:</string>
       </property>
       <property name="buddy" >
        <cstring>compositingType</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="compositingType" >
       <item>
        <property name="text" >
         <string>OpenGL</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>XRender</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Keep window thumbnails:</string>
       </property>
       <property name="buddy" >
        <cstring>windowThumbnails</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QComboBox" name="windowThumbnails" >
       <item>
        <property name="text" >
         <string>Always</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Only for shown windows</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Never</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="disableChecks" >
     <property name="whatsThis" >
      <string>Enabling this option allows compositing to be activated even if some of the internal checks fail. Doing so may make the whole desktop unusable and its use is not recommened. Use only if KWin refuses to activate compositing on a system that should be capable of compositing.
</string>
     </property>
     <property name="text" >
      <string>Disable functionality checks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="glGroup" >
     <property name="title" >
      <string>OpenGL Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QFormLayout" name="formLayout_2" >
        <property name="labelAlignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>OpenGL mode:</string>
          </property>
          <property name="buddy" >
           <cstring>glMode</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="glMode" >
          <item>
           <property name="text" >
            <string>Texture From Pixmap</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Shared Memory</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Fallback</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Texture filter:</string>
          </property>
          <property name="buddy" >
           <cstring>glTextureFilter</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="glTextureFilter" >
          <item>
           <property name="text" >
            <string>Nearest (fastest)</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Bilinear</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Trilinear (best quality)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="glDirect" >
        <property name="text" >
         <string>Direct rendering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="glVSync" >
        <property name="text" >
         <string>Use VSync</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="xrenderGroup" >
     <property name="title" >
      <string>XRender Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="xrenderSmoothScale" >
        <property name="text" >
         <string>Smooth scaling (slower)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>glGroup</zorder>
  <zorder>xrenderGroup</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>disableChecks</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>disableChecks</zorder>
  <zorder>disableChecks</zorder>
 </widget>
 <tabstops>
  <tabstop>compositingType</tabstop>
  <tabstop>windowThumbnails</tabstop>
  <tabstop>disableChecks</tabstop>
  <tabstop>glMode</tabstop>
  <tabstop>glTextureFilter</tabstop>
  <tabstop>glDirect</tabstop>
  <tabstop>glVSync</tabstop>
  <tabstop>xrenderSmoothScale</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
